servers/jax-rs-tests/src/test/java/org/projectnessie/jaxrs/TestJerseyRestNaiveClientInMemory.java

```diff
@@ -48,7 +49,7 @@
 class TestJerseyRestNaiveClientInMemory extends AbstractTestJerseyRest {
 
   @Override
-  protected void init(NessieApiV1 api, @Nullable HttpClient.Builder httpClient) {
+  protected void init(NessieApiV1 api, @Nullable HttpClient.Builder httpClient, URI uri) {
```

Comment:
this URI configuration will be saved and used from testcase to build the client (for example, at spark executor )